code/controllers/subsystem/SSambience.dm

```diff
@@ -10,6 +10,10 @@ SUBSYSTEM_DEF(ambience)
 	var/list/ambience_listening_clients = list()
 
 /datum/controller/subsystem/ambience/fire(resumed)
+	if(GLOB.configuration.general.disable_ambient_noise)
+		flags |= SS_NO_FIRE
+		return
+
 	for(var/C in ambience_listening_clients)
```

Comment:
If anyone has a better recommendation for how to handle this, I would appreciate it.

Reply:
@AffectedArc07 sanity check for this please?